app/models/ability.rb

```diff
@@ -31,6 +36,7 @@ def initialize(user, service: false)
     project_ids = projects.pluck(:id)
     can %i[read add_to], Project, id: project_ids
     can :manage, Workspace, project: { id: project_ids }
+    cannot :feature, Workspace # only site admins can feature workspaces
```

Comment:
We've painted ourselves into a corner on this one by making `featured` a property of the workspace, and then restricting it just to site admins. CanCanCan frowns on `cannot`, but it seems like it works well enough until we're sure what we want out of that home page. 

In the future, maybe we extract the featured flag to its own model with some curation tools and this unpleasantness all goes away (after refactoring the world..)

Reply:
@cbeer Here's a question I didn't want to ask, but... if an admin features a (public) workspace (within a public collection), and then one or another user flips either that workspace or its collection to non-public... what should we do? Nothing? Should we have safeguards against this or user-facing notifications?

cc: @ggeisler @camillevilla

Reply:
It won't display on the page because we're already filtering based on publicly visible workspaces. At least for now, I don't think it's worth trying to do anything more clever (like toggling `featured` off, warning anyone, etc) 🤷‍♂️

Reply:
In the longer run (could be sooner; if we want to start thinking about other potential site admin features that might be useful) maybe if we control the featuring of workspaces through an admin-only page (i.e., move the "Feature workspace" control out of the workspace details page and into a new admin-only page that lists all workspace eligible for featuring, with a quick affordance to feature/unfeature a given workspace directly), this can be handled a bit better? I think I hinted at that in one of my tickets, because at least from a UX point of view that would make it a lot easier for the admin to feature/unfeature workspaces. 

(This might be what @cbeer is suggesting with "maybe we extract the featured flag to its own model with some curation tools".)

Reply:
Yeah, I'm less worried about continuing to display them on the explore page and more wondering whether we ought to let users know the consequences of their actions. But also... I buy that now isn't the right time to deal with this complication since the logic of what to show (and not show) on the explore page is simple enough to figure out what's going on.

Reply:
Yeah, it's definitely a good point @mjgiarlo. And now that I think about it, I realize my suggestion doesn't really address the main issue you're raising (it would still enable a scenario where the admin has featured a workspace that then becomes ineligible for featuring). But, yeah, probably not a big enough thing to worry about at this stage of things...